Source/DataSources/KmlDataSource.js

```diff
@@ -2575,6 +2597,15 @@ define([
                 var newEntity = newEntities[i];
                 if (!defined(newEntity.parent)) {
                     newEntity.parent = networkLinkEntity;
+
+                    if (defined(networkLinkAvailability)) {
```

Comment:
@tfili  You have essentially the same block of code copy and pasted 3 times, can you please turn this into a helper function instead.